app/ibctesting/simapp/app.go

```diff
@@ -378,13 +381,21 @@ func NewSimApp(
 
 	app.FeeGrantKeeper = feegrantkeeper.NewKeeper(appCodec, keys[feegrant.StoreKey], app.AccountKeeper)
 	app.UpgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath, app.BaseApp, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+	app.IbcTransferMiddlewareKeeper = ibctransfermiddleware.NewKeeper(appCodec, keys[ibctransfermiddlewaretypes.StoreKey], authtypes.NewModuleAddress(govtypes.ModuleName).String(),
+		[]string{
```

Comment:
what's this?

Reply:
i think this is for setting fees. this is for testnet

Reply:
so it's an unrelated change?

Reply:
this needed to make tests build success.

Reply:
we introduce keeper to ibc hook
so we need to pass it now in tests.
this i why i added a IbcTransferMiddlewareKeeper to pass into ibc hook